src/coach/OffensiveCoach.cpp

```diff
@@ -212,7 +214,7 @@ std::pair<Vector2,bool> OffensiveCoach::penaltyAim(const Vector2 &fromPoint, dou
 }
 std::pair<Vector2, Vector2> OffensiveCoach::getAimPoints(const Vector2 &fromPoint) {
     std::pair<Vector2, Vector2> goalSides = world::field->getGoalSides(false);
-    double angleMargin = sin(2.0/180.0*M_PI);
+    double angleMargin = sin(2.0/180.0*M_PI); // (2 pi)/180 ?? why
```

Comment:
We give a small error correction of 2 degrees inwards when aiming, since aim inaccuracy obviously grows worse with shooting distance.